Dockerfile

```diff
@@ -98,9 +141,8 @@ RUN apt update -qq && apt install -qq -y --no-install-recommends \
     && yes | sdkmanager "platform-tools" \
     && yes | sdkmanager "emulator" \
     && yes | sdkmanager \
-        "platforms;android-29" \
-        "build-tools;29.0.2" \
-        "build-tools;28.0.3" \
+        "platforms;android-$ANDROID_BUILD_VERSION" \
+        "build-tools;$ANDROID_TOOLS_VERSION" \
         "cmake;$CMAKE_VERSION" \
         "system-images;android-21;google_apis;armeabi-v7a" \
         "system-images;android-28;default;x86_64" \
```

Comment:
do we have to update the version here or is it okay to keep this as is? guessing this corresponds to the version used for creating the device emulator, so in theory this should be able to work despite changing the build tools + platform number?